source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -392,19 +389,16 @@ def get_exclusion_areas(self, request):
                     )
             return {'avoid': boxes}
 
-    def _get_language(self, language):
-        try:
-            return Languages[language]
-        except KeyError:
-            self.log.error('Here parameters language={} not exist - force to english'.format(language))
-            return Languages.english
+    def _get_language(self, language_value):
+        language = LANGUAGE_TRANSFORMATION_LIST.get(language_value)
+        if not language:
+            self.log.error('Here parameter language={} Invalid - fallback to english'.format(language_value))
+            language = "en-EN"
+        return language
 
     def get_language_parameter(self, request):
-        _language = request.get('_here_language', None)
-        if _language == None:
-            return self.language
-        else:
-            return self._get_language(_language.lower())
+        language = request.get('language', 'fr-FR')
+        return self.language if not language else self._get_language(language)
```

Comment:
It looks like you deal with the default value in too many places, so it ends up being inconsistent between fr-Fr en en-US.
I'm suggesting using `None` most of the time, and default in `None` case only at the end, once and for all.

Here, the seconf line is useless if you have a default in first one, I think:
```suggestion
        language = request.get('language', None)
        return self.language if not language else self._get_language(language)
```

Reply:
:+1: